### ISSUE TITLE
Test restore - March 2024

### ISSUE THREAD

**Issue by tms-epcc** [wfau/ScienceArchives#478]
Once this is done - create new ticket for the next test restore in the next quarter.
Will occasionally ask Mike or Eckhard to do it to remind them how to do it

**Comment by tms-epcc** [comment]
29/MAR/24
- @dr-bob aiming to do this soon

**Comment by tms-epcc** [comment]
29/MAY/24
- test restore executed 
- all went well 
- no issues arising
